src/Commands/SpalpCommands.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\spalp\Commands;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\spalp\Service\Core;
+use Drupal\spalp\Service\SpalpConfig;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Drush commands for spalp.
+ */
+class SpalpCommands extends DrushCommands {
+
+  /**
+   * Spalp Core Service.
+   *
+   * @var \Drupal\spalp\Service\Core
+   */
+  protected $spalpCore;
+
+  /**
+   * Spalp Config Service.
+   *
+   * @var \Drupal\spalp\Service\SpalpConfig
+   */
+  private $spalpConfig;
+
+  /**
+   * SpalpCommands constructor.
+   *
+   * @param \Drupal\spalp\Service\Core $spalpCore
+   *   Spalp Core Service.
+   * @param \Drupal\spalp\Service\SpalpConfig $spalpConfig
+   *   Spalp Config Service.
+   */
+  public function __construct(Core $spalpCore, SpalpConfig $spalpConfig) {
+    $this->spalpCore = $spalpCore;
+    $this->spalpConfig = $spalpConfig;
+  }
+
+  /**
+   * Re-import JSON from config file to an app landing node.
+   *
+   * @param string $module
+   *   The machine name of the module.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   * @throws \Drush\Exceptions\UserAbortException
+   *
+   * @usage spalp:import-json spalp_example
+   *   Re-import JSON for the spalp_example module.
+   *
+   * @command spalp:import-json
+   * @aliases sij
+   */
+  public function importJson($module = '') {
+
+    if (empty($module)) {
+      $modules = $this->spalpConfig->getAppIds();
```

Comment:
This needs a comment on what happens if $module is empty. Not clear from the code.